### PR TITLE
split path out and remove deprecated APIs

### PR DESCRIPTION
**Pull request by loganj**

### CODE REVIEW

**Comment by rjrjr** [comment]
Needs to catch up with your method rename.

File: flow-sample/src/main/java/com/example/flow/GsonParceler.java [hunk 1]
```diff
@@ -43,7 +42,7 @@ public GsonParceler(Gson gson) {
     }
   }
 
-  @Override public Path unwrap(Parcelable parcelable) {
+  @Override public Object unwrap(Parcelable parcelable) {
```
Comment (almozavr) [inline flow-sample/src/main/java/com/example/flow/GsonParceler.java]: Object could be <T> T so great GsonParceler could be reused by third-parties, like I do for Immutables.org: https://gist.github.com/almozavr/2aab84089b23cdf2d99d

**Comment by rjrjr** [comment]
LGTM

**Comment by rjrjr** [comment]
PR is bogus, trying to merge into loganj/names instead of master.

**Pull request closed without merging**